wgsl/index.bs

```diff
@@ -2691,20 +2714,20 @@ describe the contents of memory.
 Memory consists of a set of distinct <dfn noexport>memory locations</dfn>.
 Each memory location is 8-bits in size.
 An operation affecting memory interacts with a set of one or more memory locations.
-Memory operations on [=composites=] [=behavioral requirement|will=] not
+Memory operations on [=composites=] [=behavioral requirement|will not=]
 access padding memory locations.
 
 Two sets of memory locations <dfn noexport>overlap</dfn> if the intersection of
 their sets of memory locations is non-empty.
 
 ### Memory Access Mode ### {#memory-access-mode}
 
-A <dfn noexport>memory access</dfn> is an operation that acts on memory locations.
+A <dfn noexport>memory access</dfn> is an operation that acts on [=memory locations=].
 
 * A <dfn noexport>read access</dfn> observes the contents of memory locations.
 * A <dfn noexport>write access</dfn> sets the contents of memory locations.
 
-A single operation can read, write, or both read and write.
+A single operation can be a read access and/or a write access.
```

Comment:
I find the original wording more clear.  The and/or construction is  a bit wobbly for a spec, in my view.